tools/notification-configuration/identity-resolver/Program.cs

```diff
@@ -51,22 +54,27 @@ string targetvar
                         loggerFactory.CreateLogger<GitHubNameResolver>()
                     );
 
-                    var result = await githubNameResolver.GetMappingInformationFromAADName(identity);
+                    var result = await githubNameResolver.GetMappingInformationFromAADName(identityName);
+
+                    if (result == default(IdentityDetail))
```

Comment:
```suggestion
                    if (result == default(IdentityDetail) && identityEmail != null)
```

Reply:
What value does the comparison to null add?